examples/src/main/scala/example/Http4sExample.scala

```diff
@@ -17,15 +17,19 @@
 package example
 
 import cats.effect._
+import cats.effect.syntax.all._
 import com.comcast.ip4s._
 import fs2.io.net.Network
 import org.http4s.ember.client.EmberClientBuilder
 import org.http4s.ember.server.EmberServerBuilder
 import org.http4s.implicits._
+import org.http4s.otel4s.OtelMetrics
 import org.http4s.otel4s.middleware.ClientMiddleware
 import org.http4s.otel4s.middleware.ServerMiddleware
 import org.http4s.server.Server
+import org.http4s.server.middleware.Metrics
```

Comment:
since the http4s `Metrics` is in a `middleware` package, should `OtelMetrics` be in our `middleware` package?

Reply:
Thats also a thing I took from the prometheus middleware. I don't have any preferences in that regard so I can move that if it's favourable.

Reply:
I think it would be better to move it, to reduce the risk of later name collisions in another library. thanks!

Reply:
Yes seems sensible. Done!